Add info about consolidation and profit to eth staking docs (#78)
rotki/docs: usage-guides/staking.md

![Track an ETH2 validator](/images/eth2_add_validator.png)

After adding a new validator you will see a list showing the status and balance of each validator.
Validator status may be `PENDING`, `ACTIVE`, `EXITING`, `EXITED`, or `CONSOLIDATED`.

Validators with a status of `CONSOLIDATED` have been merged with another validator.
Consolidation was introduced in Ethereum's Pectra upgrade with EIP-7251 raising the maximum effective balance from 32 ETH to 2048 ETH,
allowing multiple validators to be consolidated into one while maintaining the same total amount of staked ETH.

> Rotki detects consolidation via the execution layer consolidation request transaction.

Reviewer: Maybe link an example transaction? Or docs about what that transaction is

Author: linked an example tx.

![ETH2 validator balances](/images/eth2_validators.png)

View your ETH staking details under menu `Staking > ETH`.
ETH2 stakers can track their earnings both at current ETH prices and based on the value of daily staking rewards.

Validator profit is calculated from several different sources:

- Outstanding CL (Consensus Layer) rewards - The amount by which the validator's current balance exceeds the maximum effective balance (32 or 2048 ETH depending on if its an accumulating validator).
- CL skimming withdrawals - Profit withdrawn from the validator on the consensus layer.

>   Accumulating validators also support user requested partial withdrawals that are not profit, so rotki detects partial versus skimming withdrawals by checking if there is a corresponding execution layer withdrawal request transaction.

Reviewer: Have you defined what is an accumulating/distributing validator?

> - Block and MEV rewards - Rotki must be tracking the receiving address to count these in the validator's profit.

Reviewer: ```suggestion
- Block and MEV rewards - rotki must be tracking the receiving address to count these in the validator's profit.
```

we prefer lower case rotki

> - Exits - When a validator is exited the amount its balance exceeds what was originally deposited is counted as profit.

Reviewer: ```suggestion
- Exits - When a validator is exited the profit is calculated as the exited amount minus the deposited amount
```

I believe it is easier
